From: Jonas Meurer <mejo@debian.org>
Date: Wed, 20 Sep 2017 14:02:39 +0200
Subject: Make sphinx-build ignore .pc directory

---
 conf.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/conf.py
+++ b/conf.py
@@ -68,7 +68,7 @@ release = VERSION
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build', 'eggs', '.tox']
+exclude_patterns = ['_build', 'eggs', '.tox', '.pc']
 
 # The reST default role (used for this markup: `text`) to use for all documents.
 #default_role = None
